cxl/lib/libcxl.c

```diff
@@ -10883,3 +10883,277 @@ CXL_EXPORT int cxl_memdev_read_ddr_temp(struct cxl_memdev *memdev)
 	cxl_cmd_unref(cmd);
 	return rc;
 }
+
+#define CXL_MEM_COMMAND_ID_GET_CXL_MEMBRIDGE_ERRORS CXL_MEM_COMMAND_ID_RAW
+#define CXL_MEM_COMMAND_ID_GET_CXL_MEMBRIDGE_ERRORS_OPCODE 0xFB09
```

Comment:
FB09 used for DDR BW, changes are submitted. Please change on FW side asweel.

Reply:
Taken care in the latest patch.